dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -281,11 +311,25 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return wil::str_printf<std::wstring>(L"%ls%ls%ls", m_inlineImage.c_str(), m_heroImage.c_str(), m_appLogoOverride.c_str());
     }
 
-    std::wstring AppNotificationBuilder::GetButtons()
+    std::wstring AppNotificationBuilder::GetActions()
     {
-        if (m_buttonList.size())
+        if (m_buttonList.empty() && m_textBoxList.empty() && m_comboBoxList.empty())
```

Comment:
Instead of having this if condition here, we could remove this and do the following below.

return (result.empty()) ? {} : wil::str_printf<std::wstring>(L"<actions>%ls</actions>", result.c_str());

Reply:
done